spec/lib/uniqueness_when_changed_validator_spec.rb

```diff
@@ -3,7 +3,7 @@
   describe "#uniqueness_when_changed" do
     it "queries for a new record" do
       alert = FactoryBot.build(:miq_alert)
-      expect { alert.valid? }.to make_database_queries(:count => 1)
+      expect { alert.valid? }.to make_database_queries(:count => 2)
```

Comment:
sigh

Reply:
haha, yup... default_value_for masked changes... we can do the same but I was hoping we didn't need to.